pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -209,6 +219,43 @@ func (r *ReconcileGrafanaDashboard) importDashboard(d *i8ly.GrafanaDashboard) (r
 	return reconcile.Result{}, err
 }
 
+func (r *ReconcileGrafanaDashboard) loadDashboardFromURL(d *i8ly.GrafanaDashboard) string {
```

Comment:
Would it make sense to return `(string, error)` here?

Reply:
I had the function returning errors at first but If an error occur while downloading a dashboard it won't stop dashboards from being imported as it will use the` json` field in the CR as a fallback,  that is why I decided to just log the errors instead of return them.

I am not sure whether it's the best approach, let me know your thoughts on that.

Reply:
I'd rather return an error from this function and add an error check (and log) where it's called.